src/devtools/client/debugger/src/components/shared/Accordion.css

```diff
@@ -10,7 +10,7 @@
   display: flex;
   flex-direction: column;
   overflow-y: hidden;
-  height: 100%;
+  min-height: 100%;
```

Comment:
This does break (or at least significantly change) the behavior of the pause panel accordion.

Reply:
Production is on the right, the preview for this branch is on the left.

https://user-images.githubusercontent.com/5903784/150836580-7b24f966-6eae-4a25-83cb-b17f3e480288.mp4

Reply:
I think that the preview branch version is a bit easier to use, but also feels broken if you are expecting a true accordion. We can always add a special case to either of these to keep them working.

Reply:
Great catch, thanks! 🙌 

I'll reconsider what might be the best approach (specific selector for Layout accordion only; or something else).